packages/dds/tree/src/test/simple-tree/simpleTree.bench.ts

```diff
@@ -0,0 +1,190 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+
+import { BenchmarkType, benchmark } from "@fluid-tools/benchmark";
+import {
+	generateDeepSimpleTree,
+	generateWideSimpleTree,
+	moveWideSimpleTreeValue,
+	readDeepSimpleTree,
+	readWideSimpleTree,
+	readWideSimpleTreeValue,
+	writeDeepSimpleTree,
+	writeWideSimpleTreeNewValue,
+	type DeepTreeNode,
+	type WideTreeNode,
+} from "./benchmarkUtilities.js";
+
+// number of nodes in test for wide trees
+const nodesCountWide = [
+	[10, BenchmarkType.Measurement],
+	[100, BenchmarkType.Perspective],
+	[500, BenchmarkType.Measurement],
+];
+// number of nodes in test for deep trees
+const nodesCountDeep = [
+	[1, BenchmarkType.Measurement],
+	[10, BenchmarkType.Perspective],
+	[100, BenchmarkType.Measurement],
+];
+
+describe("SimpleTree benchmarks", () => {
+	describe("Read SimpleTree", () => {
+		const leafValue = 1;
+		for (const [numberOfNodes, benchmarkType] of nodesCountDeep) {
+			let tree: DeepTreeNode;
+			let actualDepth = 0;
+			let actualValue = 0;
+
+			benchmark({
+				type: benchmarkType,
+				title: `Deep Tree as SimpleTree: reads with ${numberOfNodes} nodes`,
+				before: () => {
+					// Setup
+					tree = generateDeepSimpleTree(numberOfNodes, leafValue);
+				},
+				benchmarkFn: () => {
+					const { depth, value } = readDeepSimpleTree(tree);
+					actualDepth = depth;
+					actualValue = value;
+				},
+				after() {
+					//  Assert read values
+					assert.equal(actualDepth, numberOfNodes);
+					assert.equal(actualValue, leafValue);
+				},
+			});
+		}
+		for (const [numberOfNodes, benchmarkType] of nodesCountWide) {
+			let tree: WideTreeNode;
+			let expected = 0;
+			let actualNodesCount = 0;
+			let actualSum = 0;
+			benchmark({
+				type: benchmarkType,
+				title: `Wide Tree as SimpleTree: reads with ${numberOfNodes} nodes`,
+				before: () => {
+					// Setup
+					tree = generateWideSimpleTree(numberOfNodes, leafValue);
+					for (let i = 0; i < numberOfNodes; i++) {
+						expected += leafValue;
+					}
```

Comment:
`const expected = numberOfNodes * leafValue;` above would be a simpler way to compute this.

Reply:
Haha. You are right!